db/seeds.rb

```diff
@@ -39,7 +39,7 @@
   room = campus.rooms.new(name:Faker::Name.name,location:Faker::Address.city, capacity: rand(10..30), picture_url: Faker::Avatar.image, events_count:0)
   room.save!
   10.times do
-    user = User.all.to_a.select{|user| user.campus.id == campus.id}.sample
+    user = User.all.to_a.select{|u| u.campus.id == campus.id}.sample
```

Comment:
I submitted this change as another pull request. As long as we merge that one first, I dont think there will be a conflict. I can always fix it if there is!